spec/dummy/app/policies/application_policy.rb

```diff
@@ -1,10 +1,10 @@
-class ApplicationPolicy
-  attr_reader :user, :record
-
-  def initialize(user, record)
-    @user = user
-    @record = record
-  end
+class ApplicationPolicy < ActionPolicy::Base
+  # attr_reader :user, :record
+  #
+  # def initialize(user, record)
+  #   @user = user
+  #   @record = record
+  # end
```

Comment:
ActionPolicy accepts initialization argument `record, **params` and I'm just using those.